scripts/install.tpl.sh

```diff
@@ -76,11 +76,6 @@ kubectl_wait "olmv1-system" "deployment/catalogd-controller-manager" "60s"
 kubectl_wait "olmv1-system" "deployment/operator-controller-controller-manager" "60s"
 
 if [[ "${install_default_catalogs}" != "false" ]]; then
-    if [[ ! -f "$default_catalogs_manifest" ]]; then
-        echo "Error: Missing required default catalogs manifest file at $default_catalogs_manifest"
-        exit 1
-    fi
```

Comment:
So no way to check if the manifests are present of not?

Reply:
I was thinking about maybe doing a curl or something. But, I don't think it adds much value, tbh.
If it's not there the error will look something like:

```
error: unable to read URL "https://github.com/operator-framework/operator-controller/releases/download/v1.2.0-rc3/default-catalogs-bob.yaml", server reported 404 Not Found, status code=404
```

Reply:
The change did not fix the issue for me, it still fails
```
mutatingwebhookconfiguration.admissionregistration.k8s.io/catalogd-mutating-webhook-configuration created
Waiting for deployment "catalogd-controller-manager" rollout to finish: 0 of 1 updated replicas are available...
Waiting for deployment "catalogd-controller-manager" rollout to finish: 0 of 1 updated replicas are available...
deployment "catalogd-controller-manager" successfully rolled out
deployment.apps/catalogd-controller-manager condition met
deployment.apps/operator-controller-controller-manager condition met
Error: Missing required default catalogs manifest file at https://github.com/operator-framework/operator-controller/releases/download/v1.2.0-rc3/default-catalogs.yaml

```

Reply:
Are you seeing this failure in the upgrade-e2e tests?
It is expected that they will fail since they install `latests` [ref](https://github.com/operator-framework/operator-controller/blob/main/Makefile#L224)